src/clj_3df/compiler.cljc

```diff
@@ -572,10 +583,10 @@
         unified     (->> (:where ir) (reduce normalize []) unify-context extract-binding)
         find-syms   (extract-find-symbols (:find ir))
         key-syms    (extract-key-symbols (:find ir))
-        projection  (->Projection unified find-syms)
+        projection  (->Projection unified (reduce-kv (fn [acc k v] (conj acc (first v))) [] (group-by name find-syms)))
```

Comment:
Should this not simply be `(distinct find-syms)`?

Reply:
Hahaha true.